src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,39 @@
-export const Tabs = () => {};
+import classNames from 'classnames';
+
+export const Tabs = ({ tabs, activeTabId, onTabSelected }) => {
+  const activeTab = tabs.find(tab => tab.id === activeTabId) || tabs[0];
+
+  const handleTabClick = id => {
+    if (id !== activeTabId) {
+      onTabSelected(id);
+    }
+  };
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => (
+            <li
+              key={tab.id}
+              className={classNames({ 'is-active': activeTabId === tab.id })}
+              data-cy="Tab"
+            >
+              <a
+                href={`#${tab.id}`}
+                data-cy="TabLink"
+                onClick={() => handleTabClick(tab.id)}
+              >
+                {tab.title}
```

Comment:
Use destructuring for `tab`
```suggestion
                {tab.title}
```